src/entities/note.cpp

```diff
@@ -2277,17 +2277,36 @@ QString Note::textToMarkdownHtml(QString str, const QString &notesPath,
        src=\")((?!file:\/\/).+)\")"),
                 "\\1file://" + windowsSlash + notePath + "/\\2\"");
     */
-
-    const QString fontString =
-        settings.value(QStringLiteral("MainWindow/noteTextView.code.font"))
+    const QString previewFontString =
+        settings.value(QStringLiteral("MainWindow/noteTextEdit.font"))
```

Comment:
If the `MainWindow/noteTextView.useEditorStyles` is false then `MainWindow/noteTextView.font` needs to be used!
And if a user overrides styles with hooks when this also will break. I don't know how to get around that.

Reply:
In 20.6.2 there will be a `Utils::Misc::previewFontString()` that should be used to get the font string.

> And if a user overrides styles with hooks then this also will break. I don't know how to get around that.

But this problem still remains.

Reply:
> In 20.6.2 there will be...

You will need to merge the current `develop` branch into your branch to see the changes.

Reply:
> And RTL (right to left) is completely broken:

How is it supposed to look like? I don't see the appeal unless the bullets would be on the right hand side.
![image](https://user-images.githubusercontent.com/6073574/83940290-cef81100-a7e3-11ea-9e07-2aa425bfc841.png)

Reply:
> How is it supposed to look like? I don't see the appeal unless the bullets would be on the right hand side.

Forget what I said about RTL, it looks almost the same in the `develop` branch.